cmd/bump.go

```diff
@@ -66,27 +67,18 @@ var bumpCmd = &cobra.Command{
 			}
 		}
 
-		headRef, err := repo.Head()
+		// (aku): We need to run this command to get the current default branch in HEAD
+		defaultBranchBytes, err := exec.Command("git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@'").Output()
```

Comment:
The problem here is if the default branch is changed on an existing repo, HEAD may still point to the old default branch. At least that's the case I'm seeing in the repo that triggered this change, where `git symbolic-ref refs/remotes/origin/HEAD` is still pointing at master even after a `git fetch` but GitHub shows the default branch as main.